diff --git a/Src/pin_config.cpp b/Src/pin_config.cpp
index 5d8e8f1..d884bc2 100644
--- a/Src/pin_config.cpp
+++ b/Src/pin_config.cpp
@@ -17,7 +17,7 @@ static PinConfig default_pin_config = {
     .pwm_b_reg = (uint16_t *) &TIM8->CCR2,
     .pwm_c_reg = (uint16_t *) &TIM8->CCR1,
     .crystal_frequency_MHz = 24,
-    .motor_encoder_reg = reinterpret_cast<volatile int32_t *>(&TIM2->CNT),
+    .motor_encoder_reg = reinterpret_cast<volatile int32_t *>(&TIM5->CNT),
 };
 
 Config::Config() {
